build.sbt

```diff
@@ -128,7 +129,13 @@ lazy val amqp = pekkoConnectorProject("amqp", "amqp", Dependencies.Amqp)
 lazy val avroparquet =
   pekkoConnectorProject("avroparquet", "avroparquet", Dependencies.AvroParquet)
 
-lazy val awslambda = pekkoConnectorProject("awslambda", "aws.lambda", Dependencies.AwsLambda)
+lazy val awsSpiPekkoHttp =
+  pekkoConnectorProject("aws-spi-pekko-http", "aws.api.pekko.http", Dependencies.AwsSpiPekkoHttp)
+    .configs(IntegrationTest)
+    .settings(Defaults.itSettings)
+
+lazy val awslambda = pekkoConnectorProject("awslambda", "aws.api.pekko.http", Dependencies.AwsLambda)
```

Comment:
Is the moduleName correct? Shoudln't be:

`pekkoConnectorProject("awslambda", "aws.lambda", Dependencies.AwsLambda)`

Reply:
thanks - I've fixed the module name now